requirements.txt

```diff
@@ -2,10 +2,9 @@ black==22.1.0
 flake8==4.0.1
 isort==5.10.1
 click==8.0.4
+numpy==1.21.2
```

Comment:
Did another version cause issues? What happened here?

Reply:
If we don't have specific version, it will pull the latest one, which is ```1.22.*```, which requires python 3.8+, so the py37 testing failed.